confgenerator/logging_receivers.go

```diff
@@ -249,6 +249,61 @@ func init() {
 	LoggingReceiverTypes.RegisterType(func() Component { return &LoggingReceiverTCP{} })
 }
 
+// A LoggingReceiverFluentForward represents the configuration for a Forward Protocol receiver.
+type LoggingReceiverFluentForward struct {
+	ConfigComponent `yaml:",inline"`
+
+	ListenHost string `yaml:"listen_host,omitempty" validate:"omitempty,ip"`
+	ListenPort uint16 `yaml:"listen_port,omitempty"`
+}
+
+func (r LoggingReceiverFluentForward) Type() string {
+	return "fluent_forward"
+}
+
+func (r LoggingReceiverFluentForward) Components(tag string) []fluentbit.Component {
+	if r.ListenHost == "" {
+		r.ListenHost = "127.0.0.1"
+	}
+	if r.ListenPort == 0 {
+		r.ListenPort = 24224
```

Comment:
Needs a follow up effort / PR to ensure that if users try to use the same port in multiple pipelines, the config errors out. Priority wise, it's P2 ish. Users today will already get an error saying the port is occupied already.

The same issue applies to TCP plugin as well.

Reply:
We have b/220128869 to keep track of it